### ISSUE TITLE
Can't edit built-in groups, roles

### ISSUE THREAD

**Issue by burner1024** [WeblateOrg/weblate#7602]
### Describe the issue

https://docs.weblate.org/no/latest/admin/access.html#automatic-group-assignments

> Automatic group assignment to Users and Viewers is always recreated when upgrading from one Weblate version to another. If you want to turn it off, set the regular expression to ^$ (which won’t match anything).

Except you can't. There's no option to edit built-in groups, nor roles.

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

1. Go to admin area.
2. Click on groups, then Users
3. It can't be edited.

### Screenshots

![Captura de pantalla de 2022-05-06 13-42-52](https://user-images.githubusercontent.com/1279852/167080553-af63a45e-d189-493b-9917-84ecef86ee72.png)
![Captura de pantalla de 2022-05-06 13-43-03](https://user-images.githubusercontent.com/1279852/167080565-d6c624d0-2fa2-41a7-86ce-8dd8d7d5df1b.png)

### How do you run Weblate?

Docker container

### Weblate versions

docker-compose 4.12.1

**Comment by burner1024** [comment]
here's some activity...

**Comment by nijel** [comment]
It behaves this way since 4ad9dc9796de9648705427fc8af58b2b9f8f82e8. Unfortunately, you can't make automatic assignment editable while not allowing editing of the group itself in the Django admin interface (or at least I don't see a way to do that).

**Comment by hippalectryon-0** [comment]
@nijel What is the current way to edit the Guests group then ? In particular, what's the new way to give permission to anonymous users ? https://docs.weblate.org/no/latest/admin/access.html#list-of-groups seems outdated

**Comment by MasterIO02** [comment]
Would love to be able to modify the default roles, because I'd like to restrict access to the source strings for any registered user.
I can't create a role without that permission and set it in the automatic group assignment, because the Users role is forced to be assigned and can't be modified. Can't disable automatic group assignment on it too to add a custom role. Can't remove it from users too.

**Comment by benq66** [comment]
Hello. I've found this issue and decided to reuse it instead of creating a new one. I'm using Weblate 4.18.2 which is running in a docker container. When I try to change anything for the default role `Users` via the Dashboard and then press Save - nothing happens. I've tried to add / remove the roles, change the languages it has access to, and also to edit the regex expression for the automatic group assignment. Neither of these changes are saved when I press the Save button, and after refreshing the page the values are back to default. Django dashboard shows that it's not possible to change anything for the `Users` group, even though it was mentioned above that some of the settings can now be changed for the default groups. Is there something I'm missing or is this functionality not working properly?

**Comment by nijel** [comment]
Django interface does not support this, use /manage/teams/ for that.

**Comment by benq66** [comment]
Apologies for not making that clear, the Dashboard that I mentioned is the /manage/teams/ page. After the changes didn't work there I checked the Django dashboard to try and apply the required changes, but that didn't work either.

**Comment by nijel** [comment]
This should work, and it works for me in both Firefox and Chrome. Do you have any browser extensions that might be affecting this? Any errors in the browser console or server log?

**Comment by benq66** [comment]
I've checked the logs for the docker container via docker-compose logs (the environment variables for logging are set to `WEBLATE_DEBUG=1`, `WEBLATE_LOGLEVEL=DEBUG`). I can see the entries about changing something for a custom group and pressing Save:

```
weblate_1   | 2023-07-15T12:36:41.045462440Z nginx stdout | ip - - [15/Jul/2023:12:36:41 +0000] "GET /teams/9/ HTTP/1.1" 200 15982 "https://weblate.address.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
weblate_1   | 2023-07-15T12:36:46.477944346Z nginx stdout | ip - - [15/Jul/2023:12:36:46 +0000] "POST /teams/9/ HTTP/1.1" 302 5 "https://weblate.address.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
weblate_1   | 2023-07-15T12:36:46.799604006Z nginx stdout | ip - - [15/Jul/2023:12:36:46 +0000] "GET /teams/9/ HTTP/1.1" 200 15977 "https://weblate.address.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
```

However, there's no POST request when I try to change the `Users` group and press Save:

```
weblate_1   | 2023-07-15T12:40:57.664818059Z nginx stdout | ip - - [15/Jul/2023:12:40:57 +0000] "GET /manage/teams/ HTTP/1.1" 200 19245 "https://weblate.address.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
weblate_1   | 2023-07-15T12:41:02.019499826Z nginx stdout | ip - - [15/Jul/2023:12:41:02 +0000] "GET /teams/3/ HTTP/1.1" 200 24342 "https://weblate.address.com" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
```

I've tried to do this operation via Chrome + incognito mode, then Edge + incognito mode. I've asked a colleague to try the same (change some stuff for Users and then Save), but he said that the button doesn't do anything for him either.

There's one thing that might be a possible cause to this - the instance we're running is not a fresh install of the latest version, we've upgraded some time ago from a version that didn't have this functionality yet (i.e. prior to v. 14.4 if I'm not mistaken). Is there some config or a database entry that we can check that maybe wasn't properly updated during the upgrade?

**Comment by benq66** [comment]
Okay, after messing up with the database for some time I was able to manually change the required values the hard way. Thank you for suggestions.

**Comment by nijel** [comment]
What you describe really looks like #8605, but that was fixed in 4.15.1.

**Comment by nijel** [comment]
Anyway, this just shows why it's bad to comment on a closed issue. The problem is entirely different (the form does not submit, vs. there is no editing interface). If you want to continue in the discussion, please open a separate issue so that the original reporters do not get notified about things they are not interested in.
